Exiled.CustomRoles/API/Features/CustomRole.cs

```diff
@@ -779,7 +786,7 @@ protected bool TryAddItem(Player player, string itemName)
             if (Enum.TryParse(itemName, out ItemType type))
             {
                 if (type.IsAmmo())
-                    player.Ammo[type] = 100;
+                    player.SetAmmo(type.GetAmmoType(), InventoryItemLoader.TryGetItem(type, out ItemBase iBase) && iBase.PickupDropModel is AmmoPickup pickup ? (ushort)pickup.MaxAmmo : (ushort)100);
```

Comment:
Why SetAmmo? Isn't it supposed to be add ammo?

Reply:
Yeah, but previous logic works same

Reply:
Possibly an unintentional mistake.

Reply:
Okay, I'll edit it

Reply:
I think `SetAmmo` is better here cuz you setting role and you specify how much ammo should has role and if it by default has ammo it need to be set to specified value